src/main/java/kostka/moviecatalog/entity/EsMovie.java

```diff
@@ -1,20 +1,20 @@
 package kostka.moviecatalog.entity;
 
 import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.CompletionField;
 import org.springframework.data.elasticsearch.annotations.Document;
-import org.springframework.data.elasticsearch.annotations.Field;
 
 
 @Document(indexName = "movie")
 public class EsMovie {
 
     @Id
     private Long id;
-    @Field
+    @CompletionField
     private String name;
-    @Field
+    @CompletionField
     private String director;
-    @Field
+    @CompletionField
```

Comment:
Jen asi pro moje info, proc si to zmenil zrovna na CompletionField?

Reply:
aha, to zustalo po tom, jak jsem zkousel, proc mi fulltext v elastiku nefunguje jak ma ... ale bez uspechu :) muzu to vratit zpatky

Reply:
Pohoda. Vubec to tu nevadi.

Jo, to bude asi spatna anotace. Podle me se neuklada field do ES jako analyzovany nebo pro full text search.
Akorat CompletionField podle me neni pro fulltext, ale pro takove ty type-ahead komponenty, kdy to hleda podle prefixu nebo tak neco.

Ale taky nevim ktera anotace je spravne.
Mozna Searchable?

Reply:
Aha, Searchable to nebude :D

Reply:
:) mam na to zalozenou issue ;) timto ukoncuju tento thread :D